.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,94 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  golangci-lint:
+    name: runner / suggester / golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: latest
+          only-new-issues: true
+
+  go-consistent:
+    name: runner / go-consistent
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - uses: actions/checkout@v2
+
+      - uses: reviewdog/action-setup@v1
+        with:
+          reviewdog_version: latest # Optional. [latest,nightly,v.X.Y.Z]
+
+      - name: Install linters
+        run: |
+          cd .github/linters
+          go get github.com/quasilyte/go-consistent
+
+      - name: Run reviewdog
+        env:
+          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          reviewdog -conf=./.reviewdog.yml -reporter=github-pr-check -runners=go-consistent -fail-on-error
+
+  gofmt:
```

Comment:
As  I know `gofmt` is a part of `golangci-lint` so maybe we could just turn it on there.